### PR TITLE
Add current state and translation to the @workflow endpoint

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by sneridagh** [comment]
@tisto tempted to add the states there too... might be that we need them at some point (workflow manager?).

File: src/plone/restapi/services/workflow/info.py [hunk 1]
```diff
@@ -68,8 +68,20 @@ def __call__(self, expand=False):
...
-            {"history": json_compatible(history), "transitions": transitions}
+            {
+                "history": json_compatible(history),
+                "transitions": transitions,
+                "state": {
+                    "id": current_state,
+                    "title": wftool.getTitleForStateOnType(
+                        current_state,
+                        self.context.portal_type,
+                    ),
```
Comment (erral) [inline src/plone/restapi/services/workflow/info.py:82]: I think that `title` should go through `context.translate` like in line 69 to return its translated title.

**Review by erral — CHANGES_REQUESTED** [review]

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by avoinea — APPROVED** [review]

**Review by erral — APPROVED** [review]

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]

**Pull request merged**